include/NeoFOAM/core/error.hpp

```diff
@@ -2,6 +2,7 @@
 // SPDX-FileCopyrightText: 2023-2024 NeoFOAM authors
 #pragma once
 
+#include <stdlib>
```

Comment:
seems not work, does it work with:
```suggestion
#include <cstdlib>
```

Reply:
Ah, I think this is a clang issue. I guess gcc ships `stdlib` and clang might ship `cstdlib`?

Reply:
Think you are right - sorry for the long wait.